website/docs/d/nsxt_edgegateway_qos_profile.html.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_nsxt_edgegateway_qos_profile"
+sidebar_current: "docs-vcd-data-source-nsxt-qos-profile"
+description: |-
+  Provides a data source to read NSX-T Edge Gateway QoS profiles. Which can be used to modify NSX-T 
+  Edge Gateway Rate Limiting (QoS) configuration.
+---
+
+# vcd\_nsxt\_edgegateway\_qos\_profile
+
+Supported in provider *v3.9+* and VCD 10.3.2+ with NSX-T.
+
+Provides a data source to read NSX-T Edge Gateway QoS profiles. Which can be used to modify NSX-T
+Edge Gateway Rate Limiting (QoS) configuration.
+
+## Example Usage
+
+```hcl
+data "vcd_nsxt_manager" "first" {
+  name = "nsxt-manager-name"
+}
+
+data "vcd_nsxt_edgegateway_qos_profile" "qos-1" {
+  nsxt_manager_id = data.nsxt_manager_id.first.id
+  name            = "qos-profile-1"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `nsxt_manager_id` - (Required) NSX-T Manager where the QoS profile is defined in (can be looked up
+using `vcd_nsxt_manager` data source)
+* `name` - (Required) QoS Profile Name
+
+
+## Attribute Reference
+
+* `description` - Description of QoS Profile
+* `committed_bandwidth` - Committed bandwith specificd in Mb/s
+* `burst_size` - Burst size defined in bytes
```

Comment:
Schema says it's in Kbps and Kbps, is there a mistype somewhere or is it intended?
```go
"committed_bandwidth": {
  Type:        schema.TypeInt,
  Computed:    true,
  Description: "Committed bandwidth in Kbps",
},
"burst_size": {
  Type:        schema.TypeInt,
  Computed:    true,
  Description: "Burst size in Kbps",
},
```

Reply:
Thanks!
Source of truth is for docs. https://developer.vmware.com/apis/vmware-cloud-director/v37.1/data-structures/GatewayQoSProfile/

Fixed schema description